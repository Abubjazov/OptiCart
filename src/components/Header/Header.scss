@import '../../variables';

.header {
	position: fixed;
	top: 0;
	background: $white;
	display: flex;
	align-items: center;
	justify-content: space-between;
	width: 100vw;
	min-height: 10vh;
	padding: 0 3vw;
	z-index: 10;

	@media (max-width: 900px) {
		min-height: 5vh;
	}

	h1 {
		font-size: 31px;
		line-height: 31px;
		font-weight: 300;
		position: relative;

		span {
			color: $main-color;
		}
	}

	.header-menu {
		ul {
			display: flex;
			width: 90px;
			justify-content: space-between;
			font-weight: 700;
			font-size: 16px;
			line-height: 16px;

			a {
				color: $black;
				text-decoration: none;
				transition: all 0.1s linear;
			}
		}
	}
}
