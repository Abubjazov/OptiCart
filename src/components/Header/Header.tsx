import './Header.scss'

export const Header = () => <div className='header'>Header</div>
