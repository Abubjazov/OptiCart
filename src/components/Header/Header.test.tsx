import { render, screen } from '@testing-library/react'
import { Header } from './Header'

test('renders learn react link', () => {
	render(<Header />)
	const linkElement = screen.getByText(/sneakers/i)
	expect(linkElement).toBeInTheDocument()
})
