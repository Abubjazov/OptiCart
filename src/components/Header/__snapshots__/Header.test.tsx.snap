// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Header should render Header 1`] = `
<DocumentFragment>
  <header
    class="header"
  >
    <h1>
      sneakers. 
      <span>
        [ ]
      </span>
    </h1>
    <nav
      class="header-menu"
    >
      <ul>
        <li>
          <a
            aria-label="Go to main page"
            class=""
            href="/optiCart/"
          >
            Main
          </a>
        </li>
        <li>
          <a
            aria-label="Go to cart page"
            class=""
            href="/optiCart/cart"
          >
            Cart
          </a>
        </li>
      </ul>
    </nav>
  </header>
</DocumentFragment>
`;
